pii.go

```diff
@@ -0,0 +1,15 @@
+package stripe
+
+import (
+  "net/url"
```

Comment:
Also, the convention in Go is to only use tabs for leading whitespace on lines. `gofmt` again will fix this for you automatically.